tests/Multipart/UploadStateTest.php

```diff
@@ -70,4 +70,60 @@ public function testSerializationWorks()
         $this->assertTrue($newState->isInitiated());
         $this->assertArrayHasKey('foo', $newState->getId());
     }
-}
+    /**
+     * @dataProvider getDisplayProgressCases
+     */
+    public function testDisplayProgressPrints(
```

Comment:
could you rename this to `testDisplayProgressPrintsProgress`? we want test names to be as descriptive as possible